.github/ISSUE_TEMPLATE/helpdesk.md

```diff
@@ -0,0 +1,82 @@
+---
+name: Help Desk Ticket
+about: Technical helpdesk tickets transferred over to github issues
+title: "[HELP] "
+labels: ["help desk", "osTicket"]
+assignees: []
+---
+
+# Required
+
+## Summary
+
+## Github Issue/Projects Fields
+
+- [ ] Populate label (e.g., bug, enhancement, etc.)
+- [ ] Populate the github projects fields (e.g., status, feature description, priority, affected branch)
+
+# Ideal
+
+## Issue Type
+
+- [ ] Technical problem/error
+- [ ] Feature request
+- [ ] Other (please specify)
+
+## Environment Details
```

Comment:
This won't be provided by all tickets, so maybe move it to "optional"?

Reply:
Fixed in 97342bd98.